tests/unit/api/test_table.py

```diff
@@ -73,11 +73,14 @@ def test_get_scd_table(saved_scd_table, snowflake_scd_table):
     assert expected_msg in str(exc.value)
 
 
-def test_get_dimension_table(saved_dimension_table, snowflake_dimension_table):
+def test_get_dimension_table(
+    saved_dimension_table, snowflake_dimension_table, mock_add_columns_attributes
+):
```

Comment:
nit: Maybe we could add the `mock_add_columns_attributes` fixture to `snowflake_dimension_table` fixture.